### PR TITLE
[pipelines] Run router container as default user

### PR DESCRIPTION
**Pull request by rahulnlb1**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] CHANGELOG.md updated
- [X] Variables and other changes are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rahulnlb1** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by rahulsadanandan — APPROVED** [review]

**Review by rimusz — APPROVED** [review]
LGTM

File: stable/pipelines/templates/pipelines-statefulset.yaml [hunk 1]
```diff
@@ -198,6 +198,8 @@ spec:
           {{- end }}
             sleep 20;
             ./pipelines-k8s;
+            echo "Changing the owner of the system.yaml to pipelines user"
+            chown 1066:1066 /opt/jfrog/pipelines/var/etc/system.yaml
```
Comment (eldada) [inline stable/pipelines/templates/pipelines-statefulset.yaml:202]: We should have the id configurable. Also need to think about openshift with its id ranged. cc @peters95

**Review by eldada — APPROVED** [review]

**Comment by chukka** [comment]
securityContext:
            allowPrivilegeEscalation: false
            runAsUser: 0

File: stable/pipelines/templates/pipelines-statefulset.yaml [hunk 2]
```diff
@@ -283,16 +285,35 @@ spec:
           ports:
             - name: http-router
               containerPort: {{ .Values.pipelines.router.internalPort }}
```
Comment (chukka) [inline stable/pipelines/templates/pipelines-statefulset.yaml:287]: @rahulnlb1  breaking the CI tests

**Pull request closed without merging**